### DISCUSSION TITLE
JWT Publisher key for HS256 in caddyfile

### DISCUSSION THREAD

**Discussion by Sheaxer** [dunglas/mercure#895] [Q&A]
Hello,

I am trying to use Mercure with Symfony 6 and generate pusblisher / subscriber jwt key using HS256 algorith. I generated the key using php function random_bytes() and encoded the result using base64. How do I use these bytes as a key for mercure itself?.

If I put the base64 encoded string into caddyfile it will just convert characters into bytes which is not the original key.

**Comment by dunglas** [comment]
Currently, you must use an ASCII key for HS256. I don't think that Caddyfile allows embedding binary.
